src/mlpack/methods/ann/layer/base_layer.hpp

```diff
@@ -168,6 +169,17 @@ template <
 using TanHLayer = BaseLayer<
     ActivationFunction, InputDataType, OutputDataType>;
 
+/**
+ * Standard SoftPlus layer.
+ */
+template <
+    class ActivationFunction = SoftplusFunction,
+    typename InputDataType = arma::mat,
+    typename OutputDataType = arma::mat
+>
+using SoftplusLayer = BaseLayer<
+    ActivationFunction, InputDataType, OutputDataType>;
```

Comment:
You are right. Thats a better way. SoftPlus functions is described in its activation function file. But I agree. I will add some comments about softplus and redirect to the file for more details.